supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,95 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+=== Technical best practices
+
+* Include links only when necessary.
+* Use hyperlinks and use them consistently.
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+* Do not use JavaScript links.
```

Comment:
I think I questioned this when we originally proposed this PR... but is it even possible to add JavaScript/Flash/iFrame links in AsciiDoc? We write our product docs in AsciiDoc, so I don't think that these are relevant to us. The examples you provide couldn't be added to AsciiDoc and be expected to work.